wazo_acceptance/helpers/pickup.py

```diff
@@ -12,3 +12,16 @@ def create(self, pickup_args):
         call_pickup = self._confd_client.call_pickups.create(pickup_args)
         self._context.add_cleanup(self._confd_client.call_pickups.delete, call_pickup['id'])
         return call_pickup
+
+    def enable_directed_extension(self, extension):
+        pickup_feature = self._confd_client.extensions_features.list(search='pickup')['items'][0]
+        new_pickup_feature = pickup_feature.copy()
+        new_pickup_feature.update({
+            'exten': '_{extension}.'.format(extension=extension),
+            'enabled': True,
+        })
+
+        with self._context.helpers.bus.wait_for_asterisk_reload(pjsip=True):
```

Comment:
```suggestion
        with self._context.helpers.bus.wait_for_asterisk_reload(dialplan=True):
```

Reply:
Yes. I also created a new helper for extension_feature that I will push soon.